pkg/pdatatest/ptracetest/traces.go

```diff
@@ -287,8 +287,7 @@ func CompareSpan(expected, actual ptrace.Span) error {
 	}
 
 	if !reflect.DeepEqual(expected.Events(), actual.Events()) {
-		errs = multierr.Append(errs, fmt.Errorf("events doesn't match expected: %v, actual: %v",
-			expected.Events(), actual.Events()))
+		errs = multierr.Append(errs, fmt.Errorf("events doesn't match"))
```

Comment:
We need to expand this comparison. I submitted another issue. Feel free to take it if you're interested https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18049

Reply:
ok, I take this @dmitryax